assets/css/main.css

```diff
@@ -1,6 +1,14 @@
 .cert-icon {
-  height: 22px;
+  margin: 2px 0 0 0 !important;
+  padding: 0 !important;
 }
 .mdi.icon {
   line-height: 1
-}
\ No newline at end of file
+}
+.pointer {
+  text-align: center;
```

Comment:
Having an alignement rule in a `pointer` class feels weird, and vuetify already has helper classes for text alignement.